instrumentation/mongo/mongo-4.0-testing/src/test/groovy/Mongo4ReactiveClientTest.groovy

```diff
@@ -293,7 +198,9 @@ class Mongo4ReactiveClientTest extends MongoBaseTest {
         "$SemanticAttributes.NET_PEER_IP.key" "127.0.0.1"
```

Comment:
Can this class use the `mongoSpan` method from the base class?

Reply:
Oops, forgot to delete, yeah!